ecms_base/themes/custom/ecms/config/optional/block.block.languageswitcher.yml

```diff
@@ -0,0 +1,27 @@
+langcode: en
+status: true
+dependencies:
+  module:
+    - language
+  theme:
+    - ecms
+id: languageswitcher
+theme: ecms
+region: language_switcher
+weight: -2
+provider: null
+plugin: 'language_block:language_interface'
+settings:
+  id: 'language_block:language_interface'
+  label: 'Language switcher'
+  provider: language
+  label_display: '0'
+visibility:
+  language:
+    id: language
+    langcodes:
+      en: en
+      es: es
```

Comment:
Should the language visibility be used here?

Reply:
This is temporary and will get overwritten once we land on a solution.